### DISCUSSION TITLE
placeholder, validationMessage 을 총점검한다

### DISCUSSION THREAD

**Discussion by bigsaigon333** [woowacourse-teams/2021-gpu-is-mine#439] [Show and tell]
모든 form의  placeholder와 validationMessage를 점검한다

**Comment by bigsaigon333** [comment]
## 로그인

![로그인 폼](https://user-images.githubusercontent.com/31029000/137458403-5c68fa63-de84-4f8e-bbcd-e7af76ba5613.png)

**Reply by bigsaigon333** [reply]
## 이메일 placeholder
1안: example@gmail.com 
2안: email@email.com

**Reply by ecsimsw** [reply]
2안 한표

**Reply by sjpark-dev** [reply]
저도 2안이 좋아요,

2안에서 조금 변경할 수 있다면,
example@email.com
도 괜찮을 것 같아요!

**Comment by bigsaigon333** [comment]
## 회원가입

![회원가입 폼](https://user-images.githubusercontent.com/31029000/137458455-10f2088c-8cef-4186-a4a8-26269eb236d7.png)

**Reply by bigsaigon333** [reply]
## 이메일 placeholder
- 로그인 폼과 일치시킬건지?

## 이름 placeholder
- 뭘로 할까요?

**Reply by ecsimsw** [reply]
일치, 이름의 placeholder 불필요 한표

**Reply by sjpark-dev** [reply]
코기 의견에 동의합니다.

**Comment by bigsaigon333** [comment]
## GpuServer 등록
![GpuServer 등록 폼](https://user-images.githubusercontent.com/31029000/137458549-47babbac-83a4-4367-baab-f046ac14252d.png)

**Reply by ecsimsw** [reply]
문제없음

**Comment by bigsaigon333** [comment]
## Job 등록
![Job 등록 폼](https://user-images.githubusercontent.com/31029000/137458594-70f0f9f9-4cb1-4814-828e-24231502762c.png)

**Reply by bigsaigon333** [reply]
## Job 예상 시간
- 그냥 놔두는 건지?

## Docker Hub Url  #430 

### label
1안: Docker Hub `URL` 으로 변경
2안: Docker Hub 계정명/저장소명

### validationMessage
1안: Docker Hub `URL`을 입력해주세요
2안: Docker Hub 계정명/저장소명을 입력해주세요

**Reply by ecsimsw** [reply]
Label 기호 없음
Validation message label과 통일

**Reply by sjpark-dev** [reply]
2안/2안

**Reply by bigsaigon333** [reply]
Label: Docker Hub Image

Placeholder
Docker Hub 계정명/저장소명:태그명
hub-user/repo-name:tag

샘플 주소 밑에 쓰기
툴팁으로 태그 옵션 설명
샘플 주소 validation
